shell/common/extensions/electron_extensions_client.cc

```diff
@@ -85,7 +82,9 @@ void ElectronExtensionsClient::InitializeWebStoreUrls(
 const extensions::PermissionMessageProvider&
 ElectronExtensionsClient::GetPermissionMessageProvider() const {
   NOTIMPLEMENTED();
```

Comment:
this should be removed given we are providing an implementation.

Reply:
This matches the previous behavior.

My reading of the code was that this isn't a call we support so we do want to error out if it's reached; but since the function signature returns a reference instead of a pointer, we can't just return `nullptr` and be done with it. We have to give the compiler a reference to _something_.

Reply:
Thanks for clarifying, I am fine leaving it as such. `NOTIMPLEMENTED` is more of a soft error does not have an impact in release builds, seems like that is intended.